src/config_trapdoor.c

```diff
@@ -1268,6 +1268,50 @@ TbBool is_door_built(PlayerNumber plyr_idx, long door_idx)
     return false;
 }
 
+/**
+ * Makes all door types manufacturable.
+ */
+TbBool make_available_all_doors(PlayerNumber plyr_idx)
+{
+  SYNCDBG(0,"Starting");
+  struct Dungeon* dungeon = get_players_num_dungeon(plyr_idx);
+  if (dungeon_invalid(dungeon)) {
+      ERRORDBG(11,"Cannot do; player %d has no dungeon",(int)plyr_idx);
```

Comment:
Was this (and below) the intended log message? Looks to me you wanted to write more right?

Reply:
I don't know; I adapted the code from make_available_all_researchable_powers.